libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -39,6 +39,7 @@
 #define AP_UAVCAN_MAX_MAG_NODES 4
 #define AP_UAVCAN_MAX_BARO_NODES 4
 #define AP_UAVCAN_MAX_BI_NUMBER 4
+#define AP_UAVCAN_ESCSTATUS_MAX_NUMBER 8 // is this limited by the message size? Perhaps use existing params
```

Comment:
In UAVCAN ESC index is 5 bit which gives maximum of 32 ESCs. Maybe some changes to logging in libraries/DataFlash should be made to add ESC ID into log message?